### ISSUE TITLE
Missing pre-trained models

### ISSUE THREAD

**Issue by kberci** [hxazem/Image-Forgery-Detection-using-Deep-learning#1]
Hey,
We were trying to use your code without training and we always got the error that the pre-trained models are not found. Could you upload your .h5 files for all 3 models?
Thanks!

**Comment by HXM14** [comment]
i'm Sorry but this is My graduation project and i Finished it 3 Month ago and i lost .h5 Files For the Models You Can Train the Models by the Data sets i provide the Data sets Name in the Description and You Can use Colab Cloud service To Train the Model if You Don't have computational Resources For this Task and For Data sets you can Find it on kaggle website

    On Thursday, October 31, 2019, 6:31:15 PM GMT+2, kberci <notifications@github.com> wrote:  
 
 
Hey,
We were trying to use your code without training and we always got the error that the pre-trained models are not found. Could you upload your .h5 files for all 3 models?
Thanks!

**Comment by kberci** [comment]
Thanks for the quick reply! I think I wouldn't really go with an own training, however I'm wondering if your project is available somewhere! I would mainly be interested your literature review if you had any inspiration from some other ready to use implementations!

**Comment by HXM14** [comment]
Another projects Now, i reads papers in this and i see evaluations in Many techniques but App in production i didn't saw but there is so Many websites that Apply Techniques like this Online and use TensorFlow to train and implement Models to use it as online web Apps the reasons that there is No particular App i think that each Algorithm have a specific cases and conditions to successfully discover the Forgery in the image so there is no general Algorithms For Any case and even in one case such as copy and Move there is no General Algorithm each Algorithm have it's limitations 
   On Wednesday, November 6, 2019, 1:15:51 AM GMT+2, kberci <notifications@github.com> wrote:  
 
 
Thanks for the quick reply! I think I wouldn't really go with an own training, however I'm wondering if your project is available somewhere! I would mainly be interested your literature review if you had any inspiration from some other ready to use implementations!

**Comment by HXM14** [comment]
Train Model is easy You Can train your own Model if You want to take a path to learn the Fundamentals of Machine learning and how to Applay this in images i can help you with resources that sure can help you 

    On Wednesday, November 6, 2019, 1:48:29 AM GMT+2, x hazem <xhazem@yahoo.com> wrote:  
 
 Another projects ready to use No, i reads papers in this and i see evaluations in Many techniques but App in production i didn't saw but there is so Many websites that Apply Techniques like this Online and use TensorFlow to train and implement Models to use it as online web Apps the reasons that there is No particular App i think that each Algorithm have a specific cases and conditions to successfully discover the Forgery in the image so there is no general Algorithms For Any case and even in one case such as copy and Move there is no General Algorithm each Algorithm have it's limitations 
   On Wednesday, November 6, 2019, 1:15:51 AM GMT+2, kberci <notifications@github.com> wrote:  
 
 
Thanks for the quick reply! I think I wouldn't really go with an own training, however I'm wondering if your project is available somewhere! I would mainly be interested your literature review if you had any inspiration from some other ready to use implementations!

**Comment by pidugusundeep** [comment]
In what format should I be creating the CSV file for training?

**Comment by affanvu** [comment]
> 
> Train Model is easy You Can train your own Model if You want to take a path to learn the Fundamentals of Machine learning and how to Apply this in images i can help you with resources that sure can help you On Wednesday, November 6, 2019, 1:48:29 AM GMT+2, x hazem

Dear Hazem Mohamed,
Can you please guide me how to train models on datasets provided using colab cloud service and use these with your code.
Thanks!

**Comment by farahFif** [comment]
Hello Hazem,

I used your VGG16 code with same dataset I got training accuracy of 80% with overfitting. Is the accuracy you claim having is for training or test data ?

**Comment by Prathamesh-Jain** [comment]
> Hello Hazem,
> 
> I used your VGG16 code with same dataset I got training accuracy of 80% with overfitting. Is the accuracy you claim having is for training or test data ?

Hi Faraf,
Could you please upload the model that you trained for VGG16 here.

**Comment by HXM14** [comment]
Hello i remember this accuracy is for testing dataset and this was my
graduation project 2 years  ago i don't have the model for sorry :(
